partitioned/src/main/java/org/apache/geode/examples/partitioned/Consumer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.examples.partitioned;
+
+import java.util.*;
+import org.apache.geode.cache.client.ClientCache;
+import org.apache.geode.cache.Region;
+
+public class Consumer extends BaseClient {
+
+  public static void main(String[] args) {
+    Consumer c = new Consumer();
+    c.checkAndPrint(args);
+  }
+
+  public void checkAndPrint(String[] args) {
+    if (0 == args.length) {
+      throw new RuntimeException("Expected argument specifying region name.");
+    } else {
```

Comment:
Nitpick - no need for an else. If (0 == args.length), your method was over when it threw an exception. This could help from having a bunch of nested levels of if/else here.